resources/views/filters/save_form.blade.php

```diff
@@ -0,0 +1,46 @@
+@php
+    /** @var \App\Services\SidebarService $sidebar */
+@endphp
+@inject('sidebar', 'App\Services\SidebarService')
+
+<x-form :action="['save-filters', $campaign, $entityType, 'm' => $mode]" method="GET" id="crud-filters-form" class="block">
+<x-dialog.header>
+    {{ __('filters.actions.bookmark') }}
+</x-dialog.header>
+<article>
+    <div class="field flex flex-col gap-3 field-">    
```

Comment:
`field-` seems out of line. gap-3 also seems weird, we don't use gap-3 in any other ui. use <x-grid type="1/1"> instead of the div